File: Mobile-App/app/src/main/java/com/example/happyhabits/feature_application/feature_symptoms/domain/model/Symptom.kt
package com.example.happyhabits.feature_application.feature_symptoms.domain.model

import com.example.happyhabits.core.domain.model.Habit
import java.time.LocalDate

class Symptom (
    id: String,
    userId: String,
    date: LocalDate,
    var name: String,
    var notes : String
): Habit(id = id, userId = userId, date = date)

File: API-Server/Happy Habits App/Model/Symptom.cs
using MongoDB.Bson.Serialization.Attributes;

namespace Happy_Habits_App.Model
{
    public class Symptom(DateOnly date, string userId, string name, string notes) : Habit(date, userId)
    {
        [BsonElement("Type")]
        public string Type { get; set; } = name;
        [BsonElement("Note")]
        public string Note { get; set; } = notes;
    }
}
